<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  
  
  <title>Hexo</title>
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <meta property="og:type" content="website">
<meta property="og:title" content="Hexo">
<meta property="og:url" content="http://example.com/index.html">
<meta property="og:site_name" content="Hexo">
<meta property="og:locale" content="en_US">
<meta property="article:author" content="John Doe">
<meta name="twitter:card" content="summary">
  
    <link rel="alternate" href="/atom.xml" title="Hexo" type="application/atom+xml">
  
  
    <link rel="shortcut icon" href="/favicon.png">
  
  
  
<link rel="stylesheet" href="/css/style.css">

  
    
<link rel="stylesheet" href="/fancybox/jquery.fancybox.min.css">

  
  
</head>

<body>
  <div id="container">
    <div id="wrap">
      <header id="header">
  <div id="banner"></div>
  <div id="header-outer" class="outer">
    <div id="header-title" class="inner">
      <h1 id="logo-wrap">
        <a href="/" id="logo">Hexo</a>
      </h1>
      
    </div>
    <div id="header-inner" class="inner">
      <nav id="main-nav">
        <a id="main-nav-toggle" class="nav-icon"><span class="fa fa-bars"></span></a>
        
          <a class="main-nav-link" href="/">Home</a>
        
          <a class="main-nav-link" href="/archives">Archives</a>
        
      </nav>
      <nav id="sub-nav">
        
        
          <a class="nav-icon" href="/atom.xml" title="RSS Feed"><span class="fa fa-rss"></span></a>
        
        <a class="nav-icon nav-search-btn" title="Search"><span class="fa fa-search"></span></a>
      </nav>
      <div id="search-form-wrap">
        <form action="//google.com/search" method="get" accept-charset="UTF-8" class="search-form"><input type="search" name="q" class="search-form-input" placeholder="Search"><button type="submit" class="search-form-submit">&#xF002;</button><input type="hidden" name="sitesearch" value="http://example.com"></form>
      </div>
    </div>
  </div>
</header>

      <div class="outer">
        <section id="main">
  
    <article id="post-BMC-Genomics-基于加权基因共表达网络分析的藜麦幼苗抗涝共表达模块鉴定及核心基因挖掘" class="h-entry article article-type-post" itemprop="blogPost" itemscope itemtype="https://schema.org/BlogPosting">
  <div class="article-meta">
    <a href="/2024/08/07/BMC-Genomics-%E5%9F%BA%E4%BA%8E%E5%8A%A0%E6%9D%83%E5%9F%BA%E5%9B%A0%E5%85%B1%E8%A1%A8%E8%BE%BE%E7%BD%91%E7%BB%9C%E5%88%86%E6%9E%90%E7%9A%84%E8%97%9C%E9%BA%A6%E5%B9%BC%E8%8B%97%E6%8A%97%E6%B6%9D%E5%85%B1%E8%A1%A8%E8%BE%BE%E6%A8%A1%E5%9D%97%E9%89%B4%E5%AE%9A%E5%8F%8A%E6%A0%B8%E5%BF%83%E5%9F%BA%E5%9B%A0%E6%8C%96%E6%8E%98/" class="article-date">
  <time class="dt-published" datetime="2024-08-07T06:20:16.000Z" itemprop="datePublished">2024-08-07</time>
</a>
    
  </div>
  <div class="article-inner">
    
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="p-name article-title" href="/2024/08/07/BMC-Genomics-%E5%9F%BA%E4%BA%8E%E5%8A%A0%E6%9D%83%E5%9F%BA%E5%9B%A0%E5%85%B1%E8%A1%A8%E8%BE%BE%E7%BD%91%E7%BB%9C%E5%88%86%E6%9E%90%E7%9A%84%E8%97%9C%E9%BA%A6%E5%B9%BC%E8%8B%97%E6%8A%97%E6%B6%9D%E5%85%B1%E8%A1%A8%E8%BE%BE%E6%A8%A1%E5%9D%97%E9%89%B4%E5%AE%9A%E5%8F%8A%E6%A0%B8%E5%BF%83%E5%9F%BA%E5%9B%A0%E6%8C%96%E6%8E%98/">BMC Genomics | 基于加权基因共表达网络分析的藜麦幼苗抗涝共表达模块鉴定及核心基因挖掘</a>
    </h1>
  

      </header>
    
    <div class="e-content article-entry" itemprop="articleBody">
      
        <p>藜麦作为一种营养价值高的新兴粮食作物，近年来受到消费者的青睐;然而，洪涝作为一种非生物胁迫，严重影响了其生长发育。目前，藜麦内涝胁迫响应的分子机制缺乏相关报道;因此，通过加权基因共表达网络分析(WGCNA)探索与这些过程相关的核心基因。<br>基于转录组数据，利用WGCNA构建了一个与抗涝相关生理性状和代谢物相关的加权基因共表达网络。共获得16个密切相关的共表达模块，并从这两个模块中挖掘出10个与目标性状相关度最高的核心基因。功能注释揭示了参与涝渍胁迫的生物学过程和代谢途径，并确定了4个与抗涝性相关的候选基因，即AP2&#x2F;ERF、MYB、bHLH和WRKY家族TF。<br>这些结果为确定藜麦内涝胁迫响应的核心基因提供了线索。这可能最终为培育耐涝藜麦新品种提供理论基础。</p>
<p><img src="/images/202408072.png" alt="图片" title="文章"></p>
<p>原文信息：: <a target="_blank" rel="noopener" href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-024-10638-y">原文</a></p>
<p>发表刊物：BMC Genomics<br>是否TOP：是</p>
<p><img src="/mayi/mayi.png" alt="图片" title="蚂蚁生信"></p>
        
          <p class="article-more-link">
            <a href="/2024/08/07/BMC-Genomics-%E5%9F%BA%E4%BA%8E%E5%8A%A0%E6%9D%83%E5%9F%BA%E5%9B%A0%E5%85%B1%E8%A1%A8%E8%BE%BE%E7%BD%91%E7%BB%9C%E5%88%86%E6%9E%90%E7%9A%84%E8%97%9C%E9%BA%A6%E5%B9%BC%E8%8B%97%E6%8A%97%E6%B6%9D%E5%85%B1%E8%A1%A8%E8%BE%BE%E6%A8%A1%E5%9D%97%E9%89%B4%E5%AE%9A%E5%8F%8A%E6%A0%B8%E5%BF%83%E5%9F%BA%E5%9B%A0%E6%8C%96%E6%8E%98/#more">Read More</a>
          </p>
        
      
    </div>
    <footer class="article-footer">
      <a data-url="http://example.com/2024/08/07/BMC-Genomics-%E5%9F%BA%E4%BA%8E%E5%8A%A0%E6%9D%83%E5%9F%BA%E5%9B%A0%E5%85%B1%E8%A1%A8%E8%BE%BE%E7%BD%91%E7%BB%9C%E5%88%86%E6%9E%90%E7%9A%84%E8%97%9C%E9%BA%A6%E5%B9%BC%E8%8B%97%E6%8A%97%E6%B6%9D%E5%85%B1%E8%A1%A8%E8%BE%BE%E6%A8%A1%E5%9D%97%E9%89%B4%E5%AE%9A%E5%8F%8A%E6%A0%B8%E5%BF%83%E5%9F%BA%E5%9B%A0%E6%8C%96%E6%8E%98/" data-id="clzjgr0te000018uw1fatcw3c" data-title="BMC Genomics | 基于加权基因共表达网络分析的藜麦幼苗抗涝共表达模块鉴定及核心基因挖掘" class="article-share-link"><span class="fa fa-share">Share</span></a>
      
      
      
    </footer>
  </div>
  
</article>



  
    <article id="post-BMC-Genomics-PLEKv2-lncRNAs-and-mRNAs预测" class="h-entry article article-type-post" itemprop="blogPost" itemscope itemtype="https://schema.org/BlogPosting">
  <div class="article-meta">
    <a href="/2024/08/07/BMC-Genomics-PLEKv2-lncRNAs-and-mRNAs%E9%A2%84%E6%B5%8B/" class="article-date">
  <time class="dt-published" datetime="2024-08-07T04:38:15.000Z" itemprop="datePublished">2024-08-07</time>
</a>
    
  </div>
  <div class="article-inner">
    
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="p-name article-title" href="/2024/08/07/BMC-Genomics-PLEKv2-lncRNAs-and-mRNAs%E9%A2%84%E6%B5%8B/">BMC Genomics | PLEKv2: lncRNAs and mRNAs预测</a>
    </h1>
  

      </header>
    
    <div class="e-content article-entry" itemprop="articleBody">
      
        <p>长链非编码RNA (lncrna)是超过200个核苷酸的RNA转录物，不编码规范蛋白。它们的生物结构与信使rna (mrna)相似。为了快速准确地区分lncRNA和mRNA转录物，研究将PLEK无比对工具升级到下一个版本PLEKv2，并构建了适合动物和植物的模型。<br>PLEKv2对人类数据集的预测准确率可达98.7%。与经典工具和基于深度学习的模型相比，这比CPC2、CNCI、Wen等人的CNN、LncADeep、PLEK和NcResNet分别高出8.1%、3.7%、16.6%、1.4%、4.9%和48.9%。PLEKv2的跨种预测准确率为90%。对于灵长类动物数据集(包括黑猩猩、猕猴和大猩猩)，PLEKv2比CPC2、CNCI、LncADeep、PLEK和NcResNet更有效和稳健。此外，PLEKv2不仅适用于与人类关系密切的非人灵长类动物，也可以预测拟南芥等植物RNA序列的编码能力。<br>实验结果表明，由PLEKv2构建的模型比PLEK能够更好地区分lncrna和mrna。PLEKv2软件可在<a target="_blank" rel="noopener" href="https://sourceforge.net/projects/plek2/%E5%85%8D%E8%B4%B9%E8%8E%B7%E5%BE%97%E3%80%82">https://sourceforge.net/projects/plek2/免费获得。</a></p>
<p><img src="/images/202408071.png" alt="图片" title="文章"></p>
<p>原文信息：<a target="_blank" rel="noopener" href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-024-10662-y">https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-024-10662-y</a></p>
<p>软      件：<a target="_blank" rel="noopener" href="https://sourceforge.net/projects/plek2/">https://sourceforge.net/projects/plek2/</a></p>
<p>发表刊物：BMC Genomics</p>
<p>是否TOP：是</p>
<p><img src="/mayi/mayi.png" alt="图片" title="蚂蚁生信"></p>
        
          <p class="article-more-link">
            <a href="/2024/08/07/BMC-Genomics-PLEKv2-lncRNAs-and-mRNAs%E9%A2%84%E6%B5%8B/#more">Read More</a>
          </p>
        
      
    </div>
    <footer class="article-footer">
      <a data-url="http://example.com/2024/08/07/BMC-Genomics-PLEKv2-lncRNAs-and-mRNAs%E9%A2%84%E6%B5%8B/" data-id="clzjd1gxw0000s4uw83tv9iuy" data-title="BMC Genomics | PLEKv2: lncRNAs and mRNAs预测" class="article-share-link"><span class="fa fa-share">Share</span></a>
      
      
      
    </footer>
  </div>
  
</article>



  
    <article id="post-Paper2" class="h-entry article article-type-post" itemprop="blogPost" itemscope itemtype="https://schema.org/BlogPosting">
  <div class="article-meta">
    <a href="/2024/08/07/Paper2/" class="article-date">
  <time class="dt-published" datetime="2024-08-07T04:24:28.000Z" itemprop="datePublished">2024-08-07</time>
</a>
    
  </div>
  <div class="article-inner">
    
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="p-name article-title" href="/2024/08/07/Paper2/">Paper2</a>
    </h1>
  

      </header>
    
    <div class="e-content article-entry" itemprop="articleBody">
      
        <h1 id="HPJ-基于转录组和代谢组分析揭示了低温胁迫下不同苹果品种抗寒代谢途径的差异"><a href="#HPJ-基于转录组和代谢组分析揭示了低温胁迫下不同苹果品种抗寒代谢途径的差异" class="headerlink" title="HPJ | 基于转录组和代谢组分析揭示了低温胁迫下不同苹果品种抗寒代谢途径的差异"></a>HPJ | 基于转录组和代谢组分析揭示了低温胁迫下不同苹果品种抗寒代谢途径的差异</h1><p>冬季冻害是严重影响植物生长发育的一种重要的非生物胁迫。落叶果树通过诱导休眠来抵御冻害。然而，同一品种的不同栽培品种具有不同的抗寒策略。目前对苹果树冬季冬眠中抗冻害的分子机制知之甚少。因此，本研究以耐寒品种‘汉服’(HF)和冷敏品种‘长富士2号’(CF)的1年生枝条为材料，通过生理生化、转录组学和代谢组学分析，探讨了它们的抗寒性。结合生理生化数据，作者发现HF具有比CF更强的渗透调节能力和抗氧化酶活性，并且具有更强的抗寒性。功能富集分析表明，两个品种在冰冻胁迫下的信号转导、激素调控和糖代谢相关通路均显著富集。此外，在棉子糖家族低聚糖(RFOs)代谢途径中编码半乳糖醇合成酶、棉子糖合成酶和水苏糖合成酶的差异表达基因(DEGs)在HF中表达上调，棉子糖和水苏糖在CF中含量降低，而在HF中含量增加。HF积累4-氨基丁酸、亚精胺和抗坏血酸清除活性氧(ROS)。冷冻胁迫下，CF中氧化谷胱甘肽、维生素C、谷胱甘肽和亚精胺含量降低，清除ROS的能力较低。此外，在低温胁迫下，转录因子AP2&#x2F;乙烯响应因子(AP2&#x2F;ERF)和WRKY受到强烈诱导。综上所述，冰冻胁迫下HF和CF关键代谢组分的差异是影响其抗寒性差异的主要因素。这些研究结果加深了对苹果树在休眠期间对冻害的抗寒机制的认识。</p>
<p><img src="/images/20240807.png" alt="图片" title="文章"></p>
<p>原文信息：<a target="_blank" rel="noopener" href="https://www.sciencedirect.com/science/article/pii/S2468014122001066">https://www.sciencedirect.com/science/article/pii/S2468014122001066</a></p>
<p>发表刊物：Horticultural Plant Journal</p>
<p><img src="/mayi/mayi.png" alt="图片" title="蚂蚁生信"></p>
        
          <p class="article-more-link">
            <a href="/2024/08/07/Paper2/#more">Read More</a>
          </p>
        
      
    </div>
    <footer class="article-footer">
      <a data-url="http://example.com/2024/08/07/Paper2/" data-id="clzjcjdqf0000skuwcoqk38d7" data-title="Paper2" class="article-share-link"><span class="fa fa-share">Share</span></a>
      
      
      
    </footer>
  </div>
  
</article>



  
    <article id="post-Paper1" class="h-entry article article-type-post" itemprop="blogPost" itemscope itemtype="https://schema.org/BlogPosting">
  <div class="article-meta">
    <a href="/2024/08/07/Paper1/" class="article-date">
  <time class="dt-published" datetime="2024-08-07T04:20:34.000Z" itemprop="datePublished">2024-08-07</time>
</a>
    
  </div>
  <div class="article-inner">
    
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="p-name article-title" href="/2024/08/07/Paper1/">Paper1</a>
    </h1>
  

      </header>
    
    <div class="e-content article-entry" itemprop="articleBody">
      
        <p>BMC Genomics | 基于全长转录组的芍药种子甘油三酯合成与贮藏关键基因鉴定<br>芍药(Paeonia lactiflora Pall.)在中国被广泛种植，因为它的根被用作传统的中药，被称为“白芍”。近年来，人们发现其种子中含有丰富的不饱和脂肪酸，是一种潜在的新型油料植物。令人惊讶的是，人们对芍药的全长转录组测序知之甚少，限制了对其基因功能和分子机制的研究。<br>共获得484,931条Reads of Inserts (ROI)序列和1,455,771条全长非嵌合Reads (FLNC)序列，用于CDS预测、TF分析、SSR分析和lncRNA鉴定。此外，还进行了基因功能注释和基因结构分析。与脂质代谢生物合成途径相关的转录本共有4905个，属于28种酶。作者利用这些数据在去冗余后鉴定了10个油酸(OLE)和5个二酰基甘油酰基转移酶(DGAT)基因成员。理化性质分析和二级结构分析表明它们在基因家族中具有相似性。系统发育分析表明，OLE和DGAT家族成员的分布与拟南芥大致相同。实时荧光定量聚合酶链反应(qRT-PCR)分析显示，该基因在种子发育的不同阶段表达量均呈现先升高后降低的趋势。<br>综上所述，这些研究结果为研究芍药苗期甘油三酯(triacylglycerol, TAG)的合成和储存分子机制提供了新的思路。为选育芍药油类品种、了解芍药储油及脂质合成相关基因的功能提供理论参考。</p>
<p>原文信息：<a target="_blank" rel="noopener" href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-024-10513-w">https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-024-10513-w</a></p>
<p>发表刊物：BMC Genomics<br>是否TOP：是<br>END<br>欢迎转发朋友圈</p>
<p>如有侵犯您的版权等信息，请您及时联系我(<a href="mailto:&#x73;&#109;&#x79;&#x61;&#x6e;&#103;&#49;&#x39;&#57;&#x33;&#64;&#x66;&#111;&#x78;&#x6d;&#97;&#105;&#x6c;&#46;&#99;&#111;&#x6d;">&#x73;&#109;&#x79;&#x61;&#x6e;&#103;&#49;&#x39;&#57;&#x33;&#64;&#x66;&#111;&#x78;&#x6d;&#97;&#105;&#x6c;&#46;&#99;&#111;&#x6d;</a>)，文章翻译水平有限，还望见谅，谢谢！祝大家生活愉快，科研顺利！注：微信公众号回复“编辑邮箱”、“编辑QQ”、“群”、“编辑微信”可与编辑进行联系！谢谢大家！</p>
        
          <p class="article-more-link">
            <a href="/2024/08/07/Paper1/#more">Read More</a>
          </p>
        
      
    </div>
    <footer class="article-footer">
      <a data-url="http://example.com/2024/08/07/Paper1/" data-id="clzjce3rb0000v8uw4aks2vji" data-title="Paper1" class="article-share-link"><span class="fa fa-share">Share</span></a>
      
      
      
    </footer>
  </div>
  
</article>



  
    <article id="post-hello-world" class="h-entry article article-type-post" itemprop="blogPost" itemscope itemtype="https://schema.org/BlogPosting">
  <div class="article-meta">
    <a href="/2024/08/07/hello-world/" class="article-date">
  <time class="dt-published" datetime="2024-08-07T04:11:34.666Z" itemprop="datePublished">2024-08-07</time>
</a>
    
  </div>
  <div class="article-inner">
    
    
      <header class="article-header">
        
  
    <h1 itemprop="name">
      <a class="p-name article-title" href="/2024/08/07/hello-world/">Hello World</a>
    </h1>
  

      </header>
    
    <div class="e-content article-entry" itemprop="articleBody">
      
        <p>Welcome to <a target="_blank" rel="noopener" href="https://hexo.io/">Hexo</a>! This is your very first post. Check <a target="_blank" rel="noopener" href="https://hexo.io/docs/">documentation</a> for more info. If you get any problems when using Hexo, you can find the answer in <a target="_blank" rel="noopener" href="https://hexo.io/docs/troubleshooting.html">troubleshooting</a> or you can ask me on <a target="_blank" rel="noopener" href="https://github.com/hexojs/hexo/issues">GitHub</a>.</p>
<h2 id="Quick-Start"><a href="#Quick-Start" class="headerlink" title="Quick Start"></a>Quick Start</h2><h3 id="Create-a-new-post"><a href="#Create-a-new-post" class="headerlink" title="Create a new post"></a>Create a new post</h3><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">$ hexo new <span class="string">&quot;My New Post&quot;</span></span><br></pre></td></tr></table></figure>

<p>More info: <a target="_blank" rel="noopener" href="https://hexo.io/docs/writing.html">Writing</a></p>
<h3 id="Run-server"><a href="#Run-server" class="headerlink" title="Run server"></a>Run server</h3><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">$ hexo server</span><br></pre></td></tr></table></figure>

<p>More info: <a target="_blank" rel="noopener" href="https://hexo.io/docs/server.html">Server</a></p>
<h3 id="Generate-static-files"><a href="#Generate-static-files" class="headerlink" title="Generate static files"></a>Generate static files</h3><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">$ hexo generate</span><br></pre></td></tr></table></figure>

<p>More info: <a target="_blank" rel="noopener" href="https://hexo.io/docs/generating.html">Generating</a></p>
<h3 id="Deploy-to-remote-sites"><a href="#Deploy-to-remote-sites" class="headerlink" title="Deploy to remote sites"></a>Deploy to remote sites</h3><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">$ hexo deploy</span><br></pre></td></tr></table></figure>

<p>More info: <a target="_blank" rel="noopener" href="https://hexo.io/docs/one-command-deployment.html">Deployment</a></p>

      
    </div>
    <footer class="article-footer">
      <a data-url="http://example.com/2024/08/07/hello-world/" data-id="clzjc83e5000074uw2mln8ljo" data-title="Hello World" class="article-share-link"><span class="fa fa-share">Share</span></a>
      
      
      
    </footer>
  </div>
  
</article>



  


</section>
        
          <aside id="sidebar">
  
    

  
    

  
    
  
    
  <div class="widget-wrap">
    <h3 class="widget-title">Archives</h3>
    <div class="widget">
      <ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2024/08/">August 2024</a></li></ul>
    </div>
  </div>


  
    
  <div class="widget-wrap">
    <h3 class="widget-title">Recent Posts</h3>
    <div class="widget">
      <ul>
        
          <li>
            <a href="/2024/08/07/BMC-Genomics-%E5%9F%BA%E4%BA%8E%E5%8A%A0%E6%9D%83%E5%9F%BA%E5%9B%A0%E5%85%B1%E8%A1%A8%E8%BE%BE%E7%BD%91%E7%BB%9C%E5%88%86%E6%9E%90%E7%9A%84%E8%97%9C%E9%BA%A6%E5%B9%BC%E8%8B%97%E6%8A%97%E6%B6%9D%E5%85%B1%E8%A1%A8%E8%BE%BE%E6%A8%A1%E5%9D%97%E9%89%B4%E5%AE%9A%E5%8F%8A%E6%A0%B8%E5%BF%83%E5%9F%BA%E5%9B%A0%E6%8C%96%E6%8E%98/">BMC Genomics | 基于加权基因共表达网络分析的藜麦幼苗抗涝共表达模块鉴定及核心基因挖掘</a>
          </li>
        
          <li>
            <a href="/2024/08/07/BMC-Genomics-PLEKv2-lncRNAs-and-mRNAs%E9%A2%84%E6%B5%8B/">BMC Genomics | PLEKv2: lncRNAs and mRNAs预测</a>
          </li>
        
          <li>
            <a href="/2024/08/07/Paper2/">Paper2</a>
          </li>
        
          <li>
            <a href="/2024/08/07/Paper1/">Paper1</a>
          </li>
        
          <li>
            <a href="/2024/08/07/hello-world/">Hello World</a>
          </li>
        
      </ul>
    </div>
  </div>

  
</aside>
        
      </div>
      <footer id="footer">
  
  <div class="outer">
    <div id="footer-info" class="inner">
      
      &copy; 2024 John Doe<br>
      Powered by <a href="https://hexo.io/" target="_blank">Hexo</a>
    </div>
  </div>
</footer>

    </div>
    <nav id="mobile-nav">
  
    <a href="/" class="mobile-nav-link">Home</a>
  
    <a href="/archives" class="mobile-nav-link">Archives</a>
  
</nav>
    


<script src="/js/jquery-3.6.4.min.js"></script>



  
<script src="/fancybox/jquery.fancybox.min.js"></script>




<script src="/js/script.js"></script>





  </div>
</body>
</html>



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
  <meta name="generator" content="HTML Tidy for Mac OS X (vers 31 October 2006 - Apple Inc. build 13), see www.w3.org" />

  <title>The Sequence Manipulation Suite</title>
  <meta http-equiv="Content-Type" content="text/html; charset=us-ascii" />
  <link rel="stylesheet" href="includes/stylesheet.css" type="text/css" />
</head>

<body class="main">
  <!-- MAIN TABLE -->

  <table border="0" cellspacing="2" cellpadding="2">
    <tbody>
      <tr>
        <td class="links">
          <a href="index.html"><img src="includes/logo.png" alt="SMS logo" width="144" height="56" border="0" /></a> <!-- INSERT LINKS -->
  <div class="category">
    Format Conversion
  </div>

  <div class="program">
    <a href="combine_fasta.html" onmouseover="self.status='Combines multiple FASTA entries into a single sequence.'; return true;" onmouseout="self.status=' '; return true;">-Combine FASTA</a><br />
    <a href="embl_fasta.html" onmouseover="self.status='Returns the entire sequence contained in an EMBL file in FASTA format.'; return true;" onmouseout="self.status=' '; return true;">-EMBL to FASTA</a><br />
    <a href="embl_feat.html" onmouseover="self.status='Parses the feature table of an EMBL file and returns the feature sequences.'; return true;" onmouseout="self.status=' '; return true;">-EMBL Feature Extractor</a><br />
    <a href="embl_trans.html" onmouseover="self.status='Parses the feature table of an EMBL file and returns the protein translations.'; return true;" onmouseout="self.status=' '; return true;">-EMBL Trans Extractor</a><br />
    <a href="filter_dna.html" onmouseover="self.status='Removes non-DNA characters from text.'; return true;" onmouseout="self.status=' '; return true;">-Filter DNA</a><br />
    <a href="filter_protein.html" onmouseover="self.status='Removes non-protein characters from text.'; return true;" onmouseout="self.status=' '; return true;">-Filter Protein</a><br />
    <a href="genbank_fasta.html" onmouseover="self.status='Returns the entire sequence contained in a GenBank file in FASTA format.'; return true;" onmouseout="self.status=' '; return true;">-GenBank to FASTA</a><br />
    <a href="genbank_feat.html" onmouseover="self.status='Parses the feature table of a GenBank file and returns the feature sequences.'; return true;" onmouseout="self.status=' '; return true;">-GenBank Feature Extractor</a><br />
    <a href="genbank_trans.html" onmouseover="self.status='Parses the feature table of a GenBank file and returns the protein translations.'; return true;" onmouseout="self.status=' '; return true;">-GenBank Trans Extractor</a><br />
    <a href="one_to_three.html" onmouseover="self.status='Converts single letter amino acid codes to three letter codes.'; return true;" onmouseout="self.status=' '; return true;">-One to Three</a><br />
    <a href="range_extract_dna.html" onmouseover="self.status='Reads a list of positions and ranges and returns those parts of a DNA sequence.'; return true;" onmouseout="self.status=' '; return true;">-Range Extractor DNA</a><br />
    <a href="range_extract_protein.html" onmouseover="self.status='Reads a list of positions and ranges and returns those parts of a protein sequence.'; return true;" onmouseout="self.status=' '; return true;">-Range Extractor Protein</a><br />
    <a href="rev_comp.html" onmouseover="self.status='Determines the reverse-complement, reverse, or complement of the sequence you enter.'; return true;" onmouseout="self.status=' '; return true;">-Reverse Complement</a><br />
    <a href="split_codons.html" onmouseover="self.status='Separates bases according to codon position.'; return true;" onmouseout="self.status=' '; return true;">-Split Codons</a><br />
    <a href="split_fasta.html" onmouseover="self.status='Converts a FASTA sequence into multiple sequences.'; return true;" onmouseout="self.status=' '; return true;">-Split FASTA</a><br />
    <a href="three_to_one.html" onmouseover="self.status='Converts three letter amino acid codes to one letter codes.'; return true;" onmouseout="self.status=' '; return true;">-Three to One</a><br />
    <a href="window_extract_dna.html" onmouseover="self.status='Returns DNA sequence segments specified by a position and window size.'; return true;" onmouseout="self.status=' '; return true;">-Window Extractor DNA</a><br />
    <a href="window_extract_protein.html" onmouseover="self.status='Returns protein sequence segments specified by a position and window size.'; return true;" onmouseout="self.status=' '; return true;">-Window Extractor Protein</a>
  </div>

  <div class="category">
    Sequence Analysis
  </div>

  <div class="program">
    <a href="codon_plot.html" onmouseover="self.status='Plots codon frequency (according to the codon table you enter) for each codon in a DNA sequence.'; return true;" onmouseout="self.status=' '; return true;">-Codon Plot</a><br />
    <a href="codon_usage.html" onmouseover="self.status='Returns a standard codon usage table.'; return true;" onmouseout="self.status=' '; return true;">-Codon Usage</a><br />
    <a href="cpg_islands.html" onmouseover="self.status='Returns a list of potential CpG islands.'; return true;" onmouseout="self.status=' '; return true;">-CpG Islands</a><br />
    <a href="dna_mw.html" onmouseover="self.status='Calculates the molecular weight of DNA sequences.'; return true;" onmouseout="self.status=' '; return true;">-DNA Molecular Weight</a><br />
    <a href="dna_pattern.html" onmouseover="self.status='Returns positions of the patterns you enter.'; return true;" onmouseout="self.status=' '; return true;">-DNA Pattern Find</a><br />
    <a href="dna_stats.html" onmouseover="self.status='Returns basic sequence statistics.'; return true;" onmouseout="self.status=' '; return true;">-DNA Stats</a><br />
    <a href="fuzzy_search_dna.html" onmouseover="self.status='Returns sequences that are identical or similar to a query sequence.'; return true;" onmouseout="self.status=' '; return true;">-Fuzzy Search DNA</a><br />
    <a href="fuzzy_search_protein.html" onmouseover="self.status='Returns sequences that are identical or similar to a query sequence.'; return true;" onmouseout="self.status=' '; return true;">-Fuzzy Search Protein</a><br />
    <a href="ident_sim.html" onmouseover="self.status='Accepts aligned sequences in FASTA format and calculates the identity and similarity of each sequence pair.'; return true;" onmouseout="self.status=' '; return true;">-Ident and Sim</a><br />
    <a href="multi_rev_trans.html" onmouseover="self.status='Can be used to predict a DNA sequence in another species using a protein sequence alignment.'; return true;" onmouseout="self.status=' '; return true;">-Multi Rev Trans</a><br />
    <a href="mutate_for_digest.html" onmouseover="self.status='Finds DNA sequences that can easily be converted to a restriction site.'; return true;" onmouseout="self.status=' '; return true;">-Mutate for Digest</a><br />
    <a href="orf_find.html" onmouseover="self.status='Determines the positions of open reading frames.'; return true;" onmouseout="self.status=' '; return true;">-ORF Finder</a><br />
    <a href="pairwise_align_codons.html" onmouseover="self.status='Returns the optimal global alignment for two coding DNA sequences.'; return true;" onmouseout="self.status=' '; return true;">-Pairwise Align Codons</a><br />
    <a href="pairwise_align_dna.html" onmouseover="self.status='Returns the optimal global alignment for two DNA sequences.'; return true;" onmouseout="self.status=' '; return true;">-Pairwise Align DNA</a><br />
    <a href="pairwise_align_protein.html" onmouseover="self.status='Returns the optimal global alignment for two protein sequences.'; return true;" onmouseout="self.status=' '; return true;">-Pairwise Align Protein</a><br />
    <a href="pcr_primer_stats.html" onmouseover="self.status='Returns a report describing PCR primer properties'; return true;" onmouseout="self.status=' '; return true;">-PCR Primer Stats</a><br />
    <a href="pcr_products.html" onmouseover="self.status='Generates PCR products from a template and two primer sequences.'; return true;" onmouseout="self.status=' '; return true;">-PCR Products</a><br />
    <a href="protein_gravy.html" onmouseover="self.status='Returns the grand average of hydropathy value of protein sequences.'; return true;" onmouseout="self.status=' '; return true;">-Protein GRAVY</a><br />
    <a href="protein_iep.html" onmouseover="self.status='Returns the predicted isoelectric point of protein sequences.'; return true;" onmouseout="self.status=' '; return true;">-Protein Isoelectric Point</a><br />
    <a href="protein_mw.html" onmouseover="self.status='Calculates the molecular weight of protein sequences.'; return true;" onmouseout="self.status=' '; return true;">-Protein Molecular Weight</a><br />
    <a href="protein_pattern.html" onmouseover="self.status='Returns positions of the patterns you enter.'; return true;" onmouseout="self.status=' '; return true;">-Protein Pattern Find</a><br />
    <a href="protein_stats.html" onmouseover="self.status='Returns basic sequence statistics.'; return true;" onmouseout="self.status=' '; return true;">-Protein Stats</a><br />
    <a href="rest_digest.html" onmouseover="self.status='Converts the sequence you enter into restriction fragments.'; return true;" onmouseout="self.status=' '; return true;">-Restriction Digest</a><br />
    <a href="rest_summary.html" onmouseover="self.status='Returns the number and positions of restriction sites.'; return true;" onmouseout="self.status=' '; return true;">-Restriction Summary</a><br />
    <a href="rev_trans.html" onmouseover="self.status='Can be used to convert protein into DNA.'; return true;" onmouseout="self.status=' '; return true;">-Reverse Translate</a><br />
    <a href="translate.html" onmouseover="self.status='Returns the translation in the reading frame you specify.'; return true;" onmouseout="self.status=' '; return true;">-Translate</a>
  </div>

  <div class="category">
    Sequence Figures
  </div>

  <div class="program">
    <a href="color_align_cons.html" onmouseover="self.status='Colors a sequence alignment based on sequence conservation.'; return true;" onmouseout="self.status=' '; return true;">-Color Align Conservation</a><br />
    <a href="color_align_prop.html" onmouseover="self.status='Colors a protein alignment based on biochemical properties of residues.'; return true;" onmouseout="self.status=' '; return true;">-Color Align Properties</a><br />
    <a href="group_dna.html" onmouseover="self.status='Numbers and groups DNA according to your specifications.'; return true;" onmouseout="self.status=' '; return true;">-Group DNA</a><br />
    <a href="group_protein.html" onmouseover="self.status='Numbers and groups amino acids according to your specifications.'; return true;" onmouseout="self.status=' '; return true;">-Group Protein</a><br />
    <a href="primer_map.html" onmouseover="self.status='Shows PCR primer annealing sites, translations, and restriction sites.'; return true;" onmouseout="self.status=' '; return true;">-Primer Map</a><br />
    <a href="rest_map.html" onmouseover="self.status='Shows restriction sites and protein translations.'; return true;" onmouseout="self.status=' '; return true;">-Restriction Map</a><br />
    <a href="trans_map.html" onmouseover="self.status='Shows protein translations.'; return true;" onmouseout="self.status=' '; return true;">-Translation Map</a>
  </div>

  <div class="category">
    Random Sequences
  </div>

  <div class="program">
    <a href="mutate_dna.html" onmouseover="self.status='Introduces random mutations into DNA sequences.'; return true;" onmouseout="self.status=' '; return true;">-Mutate DNA</a><br />
    <a href="mutate_protein.html" onmouseover="self.status='Introduces random mutations into protein sequences.'; return true;" onmouseout="self.status=' '; return true;">-Mutate Protein</a><br />
    <a href="random_coding_dna.html" onmouseover="self.status='Generates a random coding sequence of the length you specify.'; return true;" onmouseout="self.status=' '; return true;">-Random Coding DNA</a><br />
    <a href="random_dna.html" onmouseover="self.status='Generates a random DNA sequence of the length you specify.'; return true;" onmouseout="self.status=' '; return true;">-Random DNA Sequence</a><br />
    <a href="random_dna_regions.html" onmouseover="self.status='Replaces regions of the DNA sequences you enter with random bases.'; return true;" onmouseout="self.status=' '; return true;">-Random DNA Regions</a><br />
    <a href="random_protein.html" onmouseover="self.status='Generates a random protein sequence of the length you specify.'; return true;" onmouseout="self.status=' '; return true;">-Random Protein Sequence</a><br />
    <a href="random_protein_regions.html" onmouseover="self.status='Replaces regions of the protein sequences you enter with random residues.'; return true;" onmouseout="self.status=' '; return true;">-Random Protein Regions</a><br />
    <a href="sample_dna.html" onmouseover="self.status='Samples bases from a DNA sequence with replacement.'; return true;" onmouseout="self.status=' '; return true;">-Sample DNA</a><br />
    <a href="sample_protein.html" onmouseover="self.status='Samples residues from a protein sequence with replacement.'; return true;" onmouseout="self.status=' '; return true;">-Sample Protein</a><br />
    <a href="shuffle_dna.html" onmouseover="self.status='Randomly shuffles the DNA sequences you enter.'; return true;" onmouseout="self.status=' '; return true;">-Shuffle DNA</a><br />
    <a href="shuffle_protein.html" onmouseover="self.status='Randomly shuffles the protein sequences you enter.'; return true;" onmouseout="self.status=' '; return true;">-Shuffle Protein</a>
  </div>

  <div class="category">
    Miscellaneous
  </div>

  <div class="program">
    <a href="iupac.html" onmouseover="self.status='IUPAC codes for DNA and protein.'; return true;" onmouseout="self.status=' '; return true;">-IUPAC codes</a><br />
    <a href="genetic_code.html" onmouseover="self.status='The genetic codes used in the Sequence Manipulation Suite.'; return true;" onmouseout="self.status=' '; return true;">-Genetic codes</a><br />
    <a href="browser_compat.html" onmouseover="self.status='Browser compatibility.'; return true;" onmouseout="self.status=' '; return true;">-Browser compatibility</a><br />
    <a href="mirror.html" onmouseover="self.status='Mirror the Sequence Manipulation Suite.'; return true;" onmouseout="self.status=' '; return true;">-Mirror this site</a><br />
    <a href="mirror.html" onmouseover="self.status='Use the Sequence Manipulation Suite off-line.'; return true;" onmouseout="self.status=' '; return true;">-Use this site off-line</a><br />
    <a href="about.html" onmouseover="self.status='About the Sequence Manipulation Suite.'; return true;" onmouseout="self.status=' '; return true;">-About this site</a><br />
    <a href="acknowledgments.html" onmouseover="self.status='Acknowledgments.'; return true;" onmouseout="self.status=' '; return true;">-Acknowledgments</a><br />
    <a href="reference.html" onmouseover="self.status='Citing the Sequence Manipulation Suite.'; return true;" onmouseout="self.status=' '; return true;">-Reference</a>
  </div>
<!-- END INSERT LINKS -->

          <div class="copyright">
            <br />
            The Sequence Manipulation Suite Copyright &copy; 2000, 2004 Paul Stothard. Send questions and comments to stothard@ualberta.ca
          </div>

          <center>
            <div class="program">
              <a href="index.html" target="_top">home</a>
            </div>
          </center>
        </td><!-- PROGRAM CONTENT -->

        <td class="program">
          <table border="0" cellspacing="0" cellpadding="2">
            <tbody>
              <tr>
                <td class="sms">Sequence Manipulation Suite:</td>
              </tr>

              <tr>
                <td class="title">Version 2</td>
              </tr>

              <tr>
                <td class="description">
                  <ul>
                    <li>The Sequence Manipulation Suite is a collection of JavaScript programs for generating, formatting, and analyzing short DNA and protein sequences. It is commonly used by molecular biologists, for teaching, and for program and algorithm testing.</li>

                    <li>See the <a href="about.html">about the Sequence Manipulation Suite</a> page for more information about individual Sequence Manipulation Suite programs.</li>

                    <li>You can easily <a href="mirror.html">mirror the Sequence Manipulation Suite</a> on your own web site, or you can use it <a href="mirror.html">off-line</a>.</li>

                    <li>This version of the Sequence Manipulation Suite represents a complete re-write of the previous version. The new version is much faster and has many new features. The <a href="http://bioinformatics.org/sms/">previous version</a> of the Sequence Manipulation Suite can still be accessed.</li>

                    <li>Send questions and comments to <a href="mailto:stothard@ualberta.ca">stothard@ualberta.ca</a>.</li>
                  </ul><br />

                  <center>
                    <a href="index.html" target="_blank">new window</a> | <a href="index.html">home</a> | <a href="reference.html">citation</a>
                  </center><span class="version">Mon Nov  6 02:56:29 2017</span>
                </td>
              </tr>

              <tr>
                <td><span class="valid">Valid XHTML 1.0; Valid CSS</span></td>
              </tr>
            </tbody>
          </table><!-- MAIN TABLE END -->
        </td>
      </tr>
    </tbody>
  </table>
</body>
</html>
